python-client/tests/llm/test_talk.py

```diff
@@ -106,11 +105,12 @@ def test_model_ask_description(german_credit_model):
     sys.version_info.minor >= 9, reason="The Langchain agent use the new ast module implemented in Python 3.8"
 )
 def test_model_create_llm_agent_invalid_version(german_credit_model, german_credit_test_data):
-    with pytest.raises(ValidationError):
+    with pytest.raises(Exception) as exc_info:
```

Comment:
I do not quietly understand this. The test is skipped in >=Python3.9 (only be run in Python3.8).
Why we still assert `"This tool relies on Python 3.9 or higher" in str(exc_info)`

Reply:
Since this code can only be run ton 3.9+, the test make sure the user in 3.8 will have a clear warning regarding the usage of this code.